code/frameworks/angular/package.json

```diff
@@ -20,18 +20,18 @@
     "url": "https://opencollective.com/storybook"
   },
   "license": "MIT",
-  "main": "dist/types/index.js",
-  "module": "dist/types/index.js",
-  "types": "dist/types/index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "files": [
     "dist/**/*",
+    "template/**/*",
     "README.md",
     "*.js",
     "*.d.ts"
   ],
   "scripts": {
-    "check": "../../../scripts/node_modules/.bin/tsc --noEmit",
-    "prep": "node ../../../scripts/prepare.js"
+    "check": "../../../scripts/node_modules/.bin/tsc",
```

Comment:
Don't need `--noEmit`?

Reply:
Ok I see why based on the `tsconfig.json`. Why do we do it this way?

Reply:
I think this way is in general better. 
You don't end up with build files when running `yarn tsc`, some tools use `tsc` directly when analyzing a whole project, which would lead to all kinds of build artifacts.